### DISCUSSION TITLE
Decouple axes guidelines, ticks, and labels

### DISCUSSION THREAD

**Discussion by Tyler-Lopez** [patrykandpatrick/vico#949] [Ideas]
Right now, axes labels are decoupled from ticks and guidelines. This is great. You can voluntarily couple them as well, but you are not forced to do so. To do so, you implement your own `HorizontalAxis.ItemPlacer` and override specifically `getLabelValues` and `getLineValues`.

However, if you want to show ticks independent of guide lines, you cannot do so. I would like to see `getLineValues` broken into `getGuideLineValues` and `getTickValues`.

**Comment by Gowsky** [comment]
Hello, @Tyler-Lopez. Could you share more details about your use case? Understanding this will help us design the API.
